internal/controllers/user_devices_controller.go

```diff
@@ -1880,6 +1882,13 @@ func (udc *UserDevicesController) PostMintDevice(c *fiber.Ctx) error {
 
 	logger.Info().Msgf("Submitted metatransaction request %s", requestID)
 
+	credentialID, err := udc.credentialSvc.CreateVinCredential(userDevice.VinIdentifier.String, makeTokenID)
```

Comment:
You don't want to do this here, nor with the id of the make.

Look [here](https://github.com/DIMO-Network/devices-api/blob/e244bec5547e79192bf9c321c6d2c5e9c86c8051/internal/services/registry/storage.go#L74).